### ISSUE TITLE
[ProStreet] Zoom in/out with keyboard doesn't work

### ISSUE THREAD

**Issue by HarGabt** [xan1242/NFS-XtendedInput#12]
Either that, or i'm missing something.

**Comment by xan1242** [comment]
Yeah unfortunately this is completely broken. Not just zooming but moving around too.

Keyboard controls aren't a high priority, this is focused mostly on gamepad support.

It's fixable as I've made it use a totally custom function (it's not polled in a standard way). [here](https://github.com/xan1242/NFS-XtendedInput/blob/ff6d0a556a22bcb94b9552689ce68dfdb47c419e/dllmain.cpp#L895)
